### PR TITLE
Create Dependabot config file

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Enable Dependabot for security notifications on dependencies https://dependabot.com/

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+- package-ecosystem: pip
```
Comment (inferno-chromium) [inline .github/dependabot.yml:3]: Why pip ?

**Review by inferno-chromium — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml:3]: Sorry I use the dependabot to generate this. I don't know what package manager the repository uses. Is this wrong?

Comment (inferno-chromium) [reply .github/dependabot.yml:3]: https://dependabot.com/docs/config-file/#package_manager-required
Should be these i think
go:modules
go:dep

Comment (naveensrinivasan) [reply .github/dependabot.yml:3]: I don't see any package manager in this repository. Am I missing something?

Comment (inferno-chromium) [reply .github/dependabot.yml:3]: dont know this. i only see docker container as package, dont see scorecard released as a go package (maybe we should). @dlorenc - can you help to review and see what is needed here.

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
Changes needed as per comment above.

File: .github/dependabot.yml (hunk 1 above)
Comment (naveensrinivasan) [reply .github/dependabot.yml:3]: Doesn’t this Python repo have dependency on pip packages?

Comment (inferno-chromium) [reply .github/dependabot.yml:3]: I didn't see the repo, got confused with scorecards, yes criticality score is all pip.

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**